src/components/LoginButton.jsx

```diff
@@ -30,11 +23,15 @@ export default function LoginButton() {
       photo={user.photoURL}
     />
   ) : (
-    <>
-      <LoginDialog open={loginDialogOpen} onClose={handleClose} />
-      <Button variant="contained" color="primary" onClick={showLoginDialog}>
-        Login
-      </Button>
-    </>
+    <Button
+      variant="contained"
+      color="primary"
+      component={Link}
+      to={{
+        state: { showLoginDialog: true },
+      }}
+      replace>
+      Login
+    </Button>
```

Comment:
This is not a comment to address in this PR, but it's interesting how simple this `Button` has become. It makes me think we should define this as `LoginButton` instead and have `HeaderNavMenu` handle the question _should I show a dropdown or a Login button?_

Reply:
Yeah I see how simple it is. We can make another PR where we move the login to the `HeaderNavMenu`